.github/workflows/release-next.yml

```
@@ -0,0 +1,133 @@
+name: Create Release of next branch
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - next
+
+jobs:
+  generate-version:
+    name: Generate Release Version
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.generate_version.outputs.version }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          ref: "next"
+
+      - name: Generate Next Release Version
+        id: generate_version
+        uses: actions/github-script@v6
+        with:
```

```suggestion
        with:
          result-encoding: string
```

default is json 😄 https://github.com/actions/github-script?tab=readme-ov-file#result-encoding

---

Does it make a difference for what I'm using? 🤔 
I'm setting the output manually from `core.setOutput` and not really reading the result.

---

this only makes sense with my other comment 😆